packages/worker/src/sdk/users/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./users"
+export * from "./events"
```

Comment:
See comments about moving some logic from controllers into sdk. Would mean we don't need to expose the user events outside of the user sdk which sounds right